papyrus-details.html

```diff
@@ -15,28 +15,35 @@
     <div id="summary-container">
       <content select="summary"/>
     </div>
-    <a id="toggle" on-tap="toggle" href="javascript:void(0)">[[_getLinkText(expanded,language)]]</a>
+    <a id="toggle" on-tap="toggle" href="javascript:void(0)">[[_getLinkText(expanded,lang)]]</a>
     <div id="expander" class$="[[_getHiddenClass(expanded)]]">
         <content/>
     </div>
   </template>
 
   <script>
-    (function(){
-        var languages = {
-            en: { open: "Open", close: "Close" },
-            fr: { open: "Ouvrir", close: "Fermer" },
-            de: { open: "Öffnen", close: "Schließen" },
-            es: { open: "Abrir", close: "Cerrar" }
-        };
+      PapyrusDetails = {};
```

Comment:
Well. Global variables are not verboten. Global variable is just an easy way to access things can be made available before any element is created, where the usage [requires it]/[would benefit from it].  
Even Polymer prescribes using global variables, not only [for behaviors](https://www.polymer-project.org/1.0/docs/devguide/behaviors#defining-behaviors), but for elements as well e.g. [to have a custom constructor](https://www.polymer-project.org/1.0/docs/devguide/registering-elements#custom-constructor).
Similarly, global variables are natural for [usages where components are declared as vanilla JS classes](https://www.polymer-project.org/blog/es6), (arguably, where Polymer is heading) since a named class declaration essentially creates a variable of that name.  Using ES6 module with the class as 'default export' could save from this, but it's only an option.

Here, the reason for the global variable is because usages, such as this [`<container-with-lang-choice>`](https://github.com/AqoviaElements/papyrus-details/pull/2/files#diff-3e22c50371c76d64ea9738dbc46a6e57R22), benefit from having available the list of supported languages. If we moved `PapyrusDetails.resources` anywhere else, like on the instance of element, we will need rewrite this pretty simple code to something harder to understand. I'm keen to see how such an implementation would stack up.